docs/pbc/all/tax-management/202410.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-taxamo-app.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Integrate the Vertex app
+description: Find out how you can integrate the Taxamo app into your Spryker shop
+draft: true
+last_updated: Jan 8, 2025
+template: howto-guide-template
+---
+
+To [install Taxamo](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/install-vertex/install-vertex.html), you need to update [spryker/tax-app:0.4.0](https://github.com/spryker/tax-app-rest-api) and install the [spryker/tax-app-rest-api](https://github.com/spryker/tax-app-rest-api) module first.
```

Comment:
Links are the same.
Provide commands to install and update the modules

Reply:
Updated